crates/oxc_codegen/tests/integration/esbuild.rs

```diff
@@ -234,6 +235,37 @@ fn test_call() {
     test_minify("(1, eval)?.(x)", "(1,eval)?.(x);");
     // testMangleMinify(t, "(1 ? eval : 2)(x)", "(0,eval)(x);");
     // testMangleMinify(t, "(1 ? eval : 2)?.(x)", "eval?.(x);");
+
+    // Webpack-specific comments
+    // TODO: Not support trailing comments yet
+    // test(
+    //     "import(// webpackFoo: 1\n // webpackBar: 2\n 'path');",
+    //     "import(\n  // webpackFoo: 1\n  // webpackBar: 2\n  \"path\"\n);\n",
+    // );
```

Comment:
Currently, We cannot support this case, because we haven't touch trailing comments

```ts
import(// webpackFoo: 1
 // webpackBar: 2
'path');
```